pkg/shell/shell.scss

```diff
@@ -127,10 +127,46 @@ html.index-page body {
         max-height: 20rem;
         overflow: auto;
     }
-}
 
-#display-language-list {
-    margin-bottom: var(--pf-global--spacer--sm);
+    .pf-c-menu {
+        // The widget needs a little spacing between the info text and itself
+        margin-top: var(--pf-global--spacer--sm);
+    }
+
+    // PF overrides to fake a multiselect widget (as one does not currently exist in PF4).
+    // A menu gives us the interaction we want, but the styling is a bit off.
+    // Therefore, we're changing the visuals here locally.
+    // PF4 upstream request for multi-select @ https://github.com/patternfly/patternfly/issues/4027
```

Comment:
Shouldn't this be in PF overwrites file where we keep all PF css hacks? We want to keep it as easy as possible to manage these hacks so having them on one place make is more transparent.

Reply:
I explained it above.

But I'll explain it here too: 

This isn't overriding PF. We're using one widget out of context, in a completely wrong way and we're doing a hack to make it look like a widget that doesn't exist.

Reply:
The issue I see with this not being in overrides, is that we can't reuse this component in other pages, since this CSS file is not shared.

Indeed we used to have till now overrides only for bug overrides, but missing components can also go in some global file? At least some CSS in pkg/lib? We can also move it in a follow up.

Reply:
Moved it to pkg/lib.

Reply:
Sure, in lib/ and we can call it ct-listselect or whatever else you prefer.